orchestration/dagster_orchestration/hca_utils/utils.py

```diff
@@ -159,72 +157,13 @@ def put_csv_in_bucket(self, local_file, target_table: str) -> str:
         return filepath
 
     # jade interactions
-    def _get_endpoint(self, endpoint: str):
-        """
-        Get relevant information, such as the HTTP method, the path, and parameters, for Jade API endpoints.
-        :param endpoint: The endpoint name to get information for.
-        :return: A dictionary containing the endpoint's method, path, and parameters.
-        """
-
-        def create_map(paths):
-            result_map = {}
-            for path, methods in paths.items():
-                for method, rest in methods.items():
-                    result_map.update(parse_data(method, rest, path))
-            return result_map
-
-        def parse_data(method, rest, path):
-            return {rest["operationId"]: {"method": method.upper(), "path": path, "params": get_params(rest)}}
-
-        def get_params(rest):
-            out = []
-            if "parameters" in rest:
-                out = [(param["name"], param["in"]) for param in rest["parameters"]]
-            return out
-
-        url = f"{self.base_url}/v2/api-docs"
-        response = Session().get(url=url)
-        paths = response.json()["paths"]
-
-        endpoint_map = create_map(paths)
-
-        try:
-            return endpoint_map[endpoint]
-        except KeyError:
-            raise KeyError(f"Endpoint named {endpoint} not found!")
-
-    def _hit_jade(self, endpoint, body=None, params=None, query=None):
-        """
-        A generic function for hitting the Jade API.
-        :param endpoint: The endpoint to interact with.
-        :param handle_ok: A function that handles the response from the server if the response is a 2xx code.
-        :param body: An optional body for the API request if needed.
-        :param params: Optional endpoint parameters (not query filters) for the API request if needed.
-        :param query: Optional query parameters for the API request if needed.
-        :return: The result of running handle_ok on the server's response, or an HTTPError.
-        """
-        ep_info = self._get_endpoint(endpoint)
-        url = f"{self.base_url}{ep_info['path']}"
-
-        if params:
-            url = url.format(**params)
-        if query:
-            url = f"{url}/?{urllib.parse.urlencode(query)}"
-
-        response = AuthorizedSession(self.gcp_creds).request(method=ep_info["method"], url=url, json=body)
-
-        if response.ok:
-            return response
-        else:
-            raise HTTPError(f"Bad response, got code of: {response.status_code} with response body {response.text}")
-
     def get_dataset_id(self) -> str:
         """
         Get the dataset ID of the provided dataset name.
         :return: The dataset id.
         """
 
-        response = self._hit_jade("enumerateDatasets", query={"filter": self.dataset})
+        response = self.data_repo_client.enumerate_datasets(filter=self.dataset)
         return response.json()["items"][0]["id"]
```

Comment:
not part of this pr,  but i don't love that the error message we'll get if we try to look up a nonexistent dataset is pretty opaque (i.e. an index error instead of a "not found" error)